phdcontrol.cpp

```diff
@@ -116,7 +117,7 @@ static wxString ReentrancyError(const char *op)
     return wxString::Format("Cannot initiate %s while %s is in progress", op, ctrl.settleOp == OP_DITHER ? "dither" : "guide");
 }
 
-bool PhdController::Guide(bool recalibrate, const SettleParams& settle, const wxRect& roi, wxString *error)
+bool PhdController::Guide(bool recalibrate, const SettleParams& settle, const wxRect& roi, bool noStickyLocking, wxString *error)

```

Comment:
let's make this a positive attribute too (and if we converge recalibrate and useStickyLock into an `options` param with option bits, then the option bit could be positive like USE_STICKY_LOCK_POS)
```suggestion
bool PhdController::Guide(bool recalibrate, const SettleParams& settle, const wxRect& roi, bool useStickyLockPos, wxString *error)
```